src/os/command/vectorlayerfillcolorcmd.js

```diff
@@ -0,0 +1,90 @@
+goog.provide('os.command.VectorLayerFillColor');
+
+goog.require('os.command.AbstractVectorStyle');
+goog.require('os.data.OSDataManager');
+goog.require('os.events.PropertyChangeEvent');
+goog.require('os.metrics');
+goog.require('os.source.PropertyChange');
+goog.require('os.ui');
+
+
+
+/**
+ * Changes the fill color of a layer
+ * @extends {os.command.AbstractVectorStyle}
+ * @param {string} layerId
+ * @param {Array<number>|string} color
+ * @param {(Array<number>|string)=} opt_oldColor
+ * @constructor
+ */
+os.command.VectorLayerFillColor = function(layerId, color, opt_oldColor) {
```

Comment:
Same goes for the vector layer versions of these commands. There is significant duplication of code happening, and it seems unnecessary.

Reply:
The vector layer commands have also been reduced from 6 down to 2.